### PR TITLE
Fix pcsx2 chain dependency

### PR DESCRIPTION
**Pull request by raelgc**
After install [`Dolphin`](https://github.com/RetroPie/RetroPie-Setup/blob/master/scriptmodules/emulators/dolphin.sh), PCSX2 will be removed, due packages dependencies.

User should be able to just reinstall PCSX2, but after request the reinstall, system will be lead to a broken dependencies state in `apt`.

The reason: Dolphin **build** requires `portaudio19-dev` and the PCSX2 package is missing a dependency on `libjack-jackd2-0:i386` (and both packages are not compatible).

This PR is just changing the PCSX2 script to explicitly require `libjack-jackd2-0:i386`, so when we request to install/reinstall, `apt` will be able to solve the dependencies.

This is valid for Ubuntu and Debian.

### CODE REVIEW

**Comment by raelgc** [comment]
I'll send a PR to PCSX2 too, maybe they can fix it directly in their dependencies.

**Comment by nlsdg** [comment]
Is this still relevant? The PR on the PCSX2 repo has been closed in september 2020 without any code change. But it mentions that portaudio is now optional.

**Comment by raelgc** [comment]
Yeah, aside this, pcsx2 has now `amd64` builds, which probably is no more conflicting with Dolphin requirements. 

TL;DR: I'll close this. If I encounter this issue again in the future, I can reopen it.

By the way, thanks for all hard work, guys.
